### PR TITLE
Reworked README, added Hosted Weblate link

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by cschramm** [comment]
I'm fine with `Bluetooth` and the weblate part but `blueman` is usually written in lowercase.

File: README.md [hunk 1]
```diff
@@ -15,37 +15,36 @@ controlling BlueZ API and simplifying bluetooth tasks such as:
...
 
 The original project page of Valmantas Palikša can be found [on Launchpad](https://launchpad.net/blueman).
-Please use the GitHub issue tracker for bugs and questions.
 
 ## Installing
 
 See [Dependencies.md](Dependencies.md) for a list of build and runtime dependencies.
 
-To install a packaged release of blueman, run `./configure && make && make install`.
+To install a packaged release of Blueman, run `./configure && make && make install`.
```
Comment (comradekingu) [inline README.md]:
```suggestion
To install a packaged release of blueman, run `./configure && make && make install`.
```

File: README.md [hunk 2]
```diff
@@ -25,27 +24,27 @@ To install a packaged release of blueman, run `./configure && make && make insta
 
 To generate and run a configure script from source, run `./autogen.sh`.
 
-If you are packaging it for your distribution please make sure to pass `--disable-schemas-compile` and run `glib-compile-schemas /datadir/glib-2.0/schemas` as part of your (un)install phase.
+If you are packaging it for your distribution, please make sure to pass `--disable-schemas-compile` and run `glib-compile-schemas /datadir/glib-2.0/schemas` as part of your (un)install phase.
 
 ## Packaged versions
 
-See the [Wiki page for information about packaged versions of blueman](https://github.com/blueman-project/blueman/wiki/Packaged-versions).
+The [wiki page has info about packaged versions of Blueman](https://github.com/blueman-project/blueman/wiki/Packaged-versions).
```
Comment (comradekingu) [inline README.md]:
```suggestion
The [wiki page has info about packaged versions of blueman](https://github.com/blueman-project/blueman/wiki/Packaged-versions).
```

File: README.md [hunk 3]
```diff
@@ -25,27 +24,27 @@ To install a packaged release of blueman, run `./configure && make && make insta
...
 ## Contributing
 
 Fork, make your changes, and issue a pull request. If you just want to edit a single file, GitHub will guide you through that process.
 
 ### Translate
 
-Translations are managed with transifex.
-Go to [transifex blueman project page](https://www.transifex.com/projects/p/MATE/resource/blueman/).
+Translations are managed on Hosted Weblate.
+Go to the [Weblate Blueman project page](https://hosted.weblate.org/projects/blueman/).
```
Comment (comradekingu) [inline README.md]:
```suggestion
Go to the [Weblate blueman project page](https://hosted.weblate.org/projects/blueman/).
```

**Comment by comradekingu** [comment]
@cschramm I did check, but got tricked by instances of "Blueman" starting sentences.
Fixed now :)

**Pull request merged**